internal/distro/rhel9/imagetype.go

```diff
@@ -214,22 +215,22 @@ func (t *imageType) initializeManifest(bp *blueprint.Blueprint,
 
 	img, err := t.image(w, t, bp.Customizations, options, packageSets, containers, rng)
 	if err != nil {
-		return nil, err
+		return nil, warnings, err
```

Comment:
I'm not sure if I missed this before, but as in other places, I think that we should return `nil` for `warnings` if there is an error. This comment applies to changes done to `initializeManifest()` in all distros.

Reply:
OK, somehow missed it :thinking: , will update in a bit